src/bug-reports/actions/get-bug-report-by-id.action.ts

```diff
@@ -8,15 +9,26 @@ export const getBugReportById = async (
   fetcher: Fetcher,
   id: string,
 ): Promise<IBugReport> => {
-  const response = await fetcher
-    .get<IBugReport>(
+  try {
+    const response = await fetcher.get<IBugReport>(
       StringUtils.bindContext(endpoints.BUG_REPORT_BY_ID, { id }),
-    )
-    .catch((_err) => {
-      throw errorFactory.create(BugReportsErr.GET_BUG_REPORT_BY_ID_FAILED);
-    });
-  if (response.status !== 200) {
-    throw errorFactory.create(BugReportsErr.GET_BUG_REPORT_BY_ID_FAILED);
+    );
+
+    switch (response.status) {
+      case 200:
+        return response.data;
+      case 403:
+        throw errorFactory.create(BugReportsErr.GET_BUG_REPORT_BY_ID_FORBIDDEN);
+      case 404:
+        throw errorFactory.create(BugReportsErr.GET_BUG_REPORT_BY_ID_NOT_FOUND);
```

Comment:
Pour s'en tenir à ce qu'on a fait sur les autres domaines il faudrait que le code d'erreur utilisé pour une 404 soit plutôt centré sur la ressource, plutôt que sur la route.

```suggestion
        throw errorFactory.create(BugReportsErr.BUG_REPORT_NOT_FOUND);
```